src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Test/ScenarioTests/AsTests.cs

```diff
@@ -37,5 +37,11 @@ public void TestNegativeAnalysisServicesServer()
         {
             NewInstance.RunPsTest(string.Format("Test-NegativeAnalysisServicesServer -location '{0}'", AsTestsBase.resourceGroupLocation));
         }
+
+        [Fact]
```

Comment:
If this test is not to be run, then it should be attributed as LiveOnly.  Otherise it should be attributed as a check-in test